pytentiostat/routines.py

```diff
@@ -38,16 +38,21 @@ def _initialize_arduino(com):
     Creates board object with Arduino(). If the connection fails it prints an error message and exits.
     Parameters
     ----------
-    com: string
         the COM port that the potentiostat is connected to.
     """
     try:
         board = Arduino(com,
                         baudrate=_BAUD_RATE)  # opens communication to Arduino
-        print("Pytentiostat connected to {}.\n".format(
-            com))
+
+
     except:
         sys.exit("Error. Could not open COM port")
+
+    if board.firmware == 'pytentiostat_firmata.ino':  # check if the board have the right firmware
+        print("Pytentiostat connected to {}.\n".format(
```

Comment:
why do you end this with a line-break?  In general we like to keep any output compact (and minimal).  Dislike chatty programs.  Prefer better designed internals....

Reply:
sorry, it was a typing mistake

Reply:
👍  no need to apologize....